# EXCEPTION CASE 25
try:
    raise
except:
    print("exception")
else:
    return
    print("no exception found")
finally:
    break
    print("final output")
